main.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"github.com/luciodesimone/golang-bootcamp/db"
+	"log"
+)
+
+func main() {
+	stout := db.Beer{
```

Comment:
following the above comment on the Beer package, a more effective way would be to have:

```
stout := beer.New( "Creamy black dense beer with coffee notes, ideal before dinner", 42, 8, 40, 92)
```